WebDriverAgentTests/IntegrationTests/XCElementSnapshotHelperTests.m

```diff
@@ -63,35 +63,35 @@ - (void)testParentMatchingType
   XCTAssertEqual(windowSnapshot.elementType, XCUIElementTypeWindow);
 }
 
-- (void)testParentMatchingOneOfTypes
+- (void)testFindVisibleParentMatchingOneOfTypes
 {
   [self goToAttributesPage];
   XCUIElement *todayPickerWheel = self.testedApplication.pickerWheels[@"Today"];
   XCTAssertTrue(todayPickerWheel.exists);
   [todayPickerWheel resolve];
-  XCElementSnapshot *datePicker = [todayPickerWheel.lastSnapshot fb_parentMatchingOneOfTypes:@[@(XCUIElementTypeDatePicker), @(XCUIElementTypeWindow)]];
```

Comment:
I thought of replacing it and thats how I wrote it earlier. But then I saw this one using same method https://github.com/facebook/WebDriverAgent/pull/276